vars/managePods.groovy

```diff
@@ -0,0 +1,59 @@
+import groovy.json.JsonSlurperClassic
+import org.folio.Constants
+import org.folio.utilities.Logger
+
+def handlePods(String clusterName, String action, String ns) {
+  buildName "${clusterName}"
+  folioHelm.withKubeConfig(clusterName) {
+    Logger logger = new Logger(this, 'managePods')
+    List namespaces = sh(script: "kubectl get namespaces -o jsonpath='{.items[*].metadata.name}'", returnStdout: true).trim().tokenize()
+    namespaces.each { namespace ->
+      if (namespace.toString().trim() in Constants.RANCHER_KNOWN_NAMESPACES) {
+        logger.info("Service namespace: ${namespace} bypassed. Skipping pods management.")
+      } else {
+        logger.info("Running in namespace: ${namespace}")
+        switch (action) {
+          case 'stop':
+            def labels = kubectl.getLabelsFromNamespace("${namespace}")
+            def status = new JsonSlurperClassic().parseText(labels)
+            if (status['suspend'] == 'yes' && ns.trim() == namespace.toString().trim()) {
+              kubectl.deleteLabelFromNamespace("${namespace}", "suspend")
+            } else {
+              def sts = sh(returnStdout: true, script: "kubectl get sts --namespace ${namespace} -o jsonpath='{.items[*].metadata.name}' --ignore-not-found").trim()
```

Comment:
Maybe better to move this call into separate method?

Reply:
Hello @OHaimanov ,
Thanks, implemented, please resolve this conversation, if all is OK